package.json

```diff
@@ -121,38 +121,38 @@
   },
   "devDependencies": {
     "@faker-js/faker": "^8.4.1",
-    "@fastify/static": "^7.0.3",
-    "@swc/core": "^1.4.14",
+    "@fastify/static": "^7.0.4",
+    "@swc/core": "^1.5.5",
     "@types/express": "^4.17.21",
     "@types/jest": "^29.5.12",
     "@types/jsonwebtoken": "^9.0.6",
-    "@types/lodash": "^4.17.0",
-    "@types/node": "^18.16.17",
+    "@types/lodash": "^4.17.1",
+    "@types/node": "^20.12.10",
```

Comment:
I usually just do `20.x`, then it's taken care of for this version

```suggestion
    "@types/node": "20.x",
```

Reply:
I don't like wildcards 😆 they can bite you really easily

Reply:
Same, but I usually find it alright for `@types/*`, as it's just types, and shouldn't cause issues™️

Reply:
But non-blocker